### ISSUE TITLE
Review section language

### ISSUE THREAD

**Issue by robredpath** [OpenDataServices/cove#593]
"The term "use the Review section below to identify potential issues." seems to be used a lot. I'm not convinced that the Review section can help me in all cases though. I can't - for example - find the dates of a grant via this. We just seem to be pointing at that section a bit more, but not really helping the user specifically."

Internal issue #7918

**Comment by edugomez** [comment]
@robredpath as I mentioned in internal issue #7918, _use the Review section below ..._ only appears once in the explore page (in 'Data Summary'). Also, it can help you in _some_ cases, eg. counting the number of grants (4) in Data Summary and then showing you in the Review section that they are in fact only two grants duplicated by mistake https://github.com/OpenDataServices/cove/blob/master/cove_360/fixtures/WellcomeTrust-grants_fixed_2_grants.json

**Comment by robredpath** [comment]
Thanks @edugomez - I think that an early version of the copy might have used the phrase more? However, I think the comment was more about the fact that we direct people to the Review section without really explaining what they should do there - 'review' doesn't really explain much if you don't really know what to do! 

@stevieflow - this was your comment originally. Now that we're a bit further down the line, do you have any further thoughts on this? Should we change it to something like "If this summary looks incorrect, you can use the Review section alongside any warnings or errors below to investigate what happened and make changes to your original data before loading the new file to check again"?

**Comment by stevieflow** [comment]
> "If this summary looks incorrect, you can use the Review section alongside any warnings or errors below to investigate what happened and make changes to your original data before loading the new file to check again"?

That could be useful - seems a bit wordy, but does say what we mean

I'd like to see this alongside #674
